ci/report-tests.ps1

```diff
@@ -75,7 +84,7 @@ $slack_attachment = [ordered]@{
                 short = $true
             }
             @{
-                value = "Passed $($test_results_obj.succeeded) / $($test_results_obj.succeeded + $test_results_obj.failed) tests."
+                value = "Passed $($test_results_obj.succeeded + $test_results_obj.succeededWithWarnings) / $num_gdk_tests tests."
```

Comment:
This line fixes a minor bug where tests that succeeded with warnings were not reported as being successfull in the slack message. 
![image](https://user-images.githubusercontent.com/54983025/69872741-34f06280-12ae-11ea-98e9-c73d88159ea3.png)